mesa/visualization/ModularVisualization.py

```diff
@@ -328,6 +327,9 @@ def __init__(
                     self.local_css_includes.add(include_file)
                 else:
                     self.local_js_includes.add(include_file)
+            if len(element.local_includes) > 0:
+                if element.local_dir != "":
+                    local_dirs.append(element.local_dir)
```

Comment:
I'm also wondering what'll happen if two origin folders with the same structure get mapped to the same destination folder.

Mesa-Geo has a `js` folder and a `css` folder under `templates` folder that maps to `/local`. Perhaps Mesa-Geo users also define their own frontend elements in the same way. Then we have another set of `js` and `css` folders mapped to `/local`. Not sure whether this will work.